docs/server-side/v2/api-reference/external-streams/stop-rtmp-for-a-given-room.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Stop RTMP Streaming and / or recording for a given room
+nav: 3.19
+---
+
+Trigger this request to stop rtmp streaming and / or recording .
+
+<PostRequest title="https://api.100ms.live/v2/external-streams/rooms/:room_id/stop" />
+<Request id="req-0">
+
+```bash
+curl --location --request POST 'https://api.100ms.live/v2/external-streams/rooms/<room_id>/stop' \
+--header 'Authorization: Bearer <management_token>' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+
+}'
+```
+
+</Request>
+<ResponseBox id="res-0" status="200 OK">
+
+```json
+{
+"error": "<error>"
+"jobs" : [{
+    "id": "<job_id>",
+    "room_id": "<room_id>>",
+    "session_id": "<session_id>",
+    "status": "<status>",
+    "started_by": "<peer_id>",
```

Comment:
Do we need a corresponding stopped_by?

Reply:
Also what is the value when it's started from the server?

Reply:
it will be empty if it is started from the server